saleor/checkout/calculations.py

```diff
@@ -1,35 +1,44 @@
-from typing import TYPE_CHECKING
+from typing import Optional, TYPE_CHECKING
 
 from ..extensions.manager import get_extensions_manager
 
 if TYPE_CHECKING:
     from prices import TaxedMoney
     from .models import Checkout, CheckoutLine
     from ..discount.types import DiscountsListType
+    from ..extensions.manager import ExtensionsManager
 
 
 def checkout_shipping_price(
-    checkout: "Checkout", discounts: "DiscountsListType" = None
+    checkout: "Checkout",
+    discounts: "DiscountsListType" = None,
+    extensions: Optional["ExtensionsManager"] = None,
```

Comment:
I am not sure about that. What was the reason to add it to all `checkout_*` methods? I don't see any benefits of using this, only additional `if`/`or`

Reply:
correct me if I'm wrong but
resolvers store extension in `info` arg,
this way we just call "get_extension" once not every time resolver is called

Reply:
@kswiatek92  See `saleor.core.middleware.extensions`. Manager is as appened as a `SimpleLazyObject` to `info.context.extensions`. So it doesn't give you any benefit to pass it as an optional argument. 
You will get the same number of `function calls` in both cases.

Reply:
in that case, your'e right I will revert this to previous version

Reply:
done